buildcontainer/Makefile

```diff
@@ -1,14 +1,18 @@
 include .env
 
 REGISTRY           ?= ghcr.io
-TAG_VERSION        ?= $(shell git describe --tags --abbrev=0)
+TAG_VERSION        ?= snapshot
+OWNER              ?= goreleaser
+GIT_REPO           ?= https://github.com/goreleaser/goreleaser-cross
```

Comment:
I _think_ we need to modify this because of the arg we're passing in now?

Reply:
Updated.